satellite/admin/README.md

```diff
@@ -4,7 +4,29 @@ Satellite Admin package provides API endpoints for administrative tasks.
 
 Requires setting `Authorization` header for requests.
 
-## GET /api/user/{user-email}
+## POST /api/user
```

Comment:
Let's make separate PR-s for each addition, that way it's clearer to see the tests etc.

Reply:
yeah, its 1200+ lines PR right now

Reply:
I'll do that.